tephra-hbase-compat-0.98/src/main/java/co/cask/tephra/hbase98/TransactionAwareHTable.java

```diff
@@ -557,15 +557,18 @@ private Delete transactionalizeAction(Delete delete) throws IOException {
 
     Map<byte[], List<Cell>> familyToDelete = delete.getFamilyCellMap();
     if (familyToDelete.isEmpty()) {
-      // perform a row delete is we are using row-level conflict detection
+      // perform a row delete if we are using row-level conflict detection
+      Result result = get(new Get(delete.getRow()));
```

Comment:
A row delete gets translated into a family delete for all families of the table. Hence, we'll have to store all families for the table in the change set, so that we can delete all the family delete markers.